lightwood/helpers/text.py

```diff
@@ -0,0 +1,27 @@
+import nltk
```

Comment:
I checked this, nltk is a dependency of lightwood

Reply:
Though its best to use this:
https://github.com/huggingface/tokenizers

And also apply it to the whole array of strings at once, not to individual phrases like now